scripts/generate-sha256.js

```diff
@@ -1,19 +1,24 @@
 #!/usr/bin/env node
-
+const path = require('path');
 const shell = require('shelljs');
 
```

Comment:
The output of this script is redirected in in the shell to a file.  I considered changing that behavior to make it less mysterious when things go wrong but decided I'll create a follow up ticket.